web/src/components/shared/ConfigureGraphs.tsx

```diff
@@ -8,6 +8,14 @@ const ConfigureGraphs = ({
   savingPromError,
   onPromValueChange,
   placeholder,
+}: {
+  toggleConfigureGraphs?: () => void;
+  updatePromValue?: () => void;
+  promValue?: string;
+  savingPromValue?: boolean;
+  savingPromError?: string;
+  onPromValueChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: string;
```

Comment:
this was causing a warning in the console when passing in an {} into an onClick handler. I saw the problem with typescript getting the implicit type and making the value required so I had to add explicit type definition to make that go away.